lib/engine/action/start_merge.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require_relative 'base'
+
+module Engine
+  module Action
+    class StartMerge < Base
+      attr_reader :corporation
+
+      def initialize(entity, corporation:)
+        @entity = entity
+        @corporation = corporation
+      end
+
+      def self.h_to_args(h, game)
+        {
+          corporation: game.corporation_by_id(h['corporation']),
+        }
+      end
+
+      def args_to_h
+        {
+          'corporation' => @corporation.id,
+        }
+      end
+    end
+  end
+end
```

Comment:
Action associated with selecting the first corporation for a merger (done during the stock round). After the corporation is selected, the game transitions to the 1817-like merge view to select the second corporation for the merger.

Reply:
do you need a new action? could you use choose?

Reply:
Makes sense. I'll use choose.